airflow/contrib/example_dags/example_gcp_vision.py

```diff
@@ -241,5 +325,84 @@
     )
     # [END howto_operator_vision_product_delete_2]
 
+    # [START howto_operator_vision_reference_image_create_2]
+    reference_image_create_2 = CloudVisionReferenceImageCreateOperator(
+        location=GCP_VISION_LOCATION,
+        reference_image=reference_image,
+        product_id=GCP_VISION_PRODUCT_ID,
+        reference_image_id=GCP_VISION_REFERENCE_IMAGE_ID,
+        retry=Retry(maximum=10.0),
+        timeout=5,
+        task_id='reference_image_create_2',
+    )
+    # [END howto_operator_vision_reference_image_create_2]
+
+    # Second 'create' task with the same product_id to demonstrate idempotence
+    reference_image_create_2_idempotence = CloudVisionReferenceImageCreateOperator(
+        location=GCP_VISION_LOCATION,
+        reference_image=reference_image,
+        product_id=GCP_VISION_PRODUCT_ID,
+        reference_image_id=GCP_VISION_REFERENCE_IMAGE_ID,
+        retry=Retry(maximum=10.0),
+        timeout=5,
+        task_id='reference_image_create_2',
+    )
+
+    # [START howto_operator_vision_add_product_to_product_set_2]
+    add_product_to_product_set_2 = CloudVisionAddProductToProductSetOperator(
+        location=GCP_VISION_LOCATION,
+        product_set_id=GCP_VISION_PRODUCT_SET_ID,
+        product_id=GCP_VISION_PRODUCT_ID,
+        retry=Retry(maximum=10.0),
+        timeout=5,
+        task_id='add_product_to_product_set_2',
+    )
+    # [END howto_operator_vision_add_product_to_product_set_2]
+
+    # [START howto_operator_vision_remove_product_from_product_set_2]
+    remove_product_from_product_set_2 = CloudVisionRemoveProductFromProductSetOperator(
+        location=GCP_VISION_LOCATION,
+        product_set_id=GCP_VISION_PRODUCT_SET_ID,
+        product_id=GCP_VISION_PRODUCT_ID,
+        retry=Retry(maximum=10.0),
+        timeout=5,
+        task_id='remove_product_from_product_set_2',
+    )
+    # [END howto_operator_vision_remove_product_from_product_set_2]
+
+    # Product path
+    product_create_2 >> product_create_2_idempotence >> product_get_2 >> product_update_2 >> product_delete_2
+
+    # ProductSet path
     product_set_create_2 >> product_set_get_2 >> product_set_update_2 >> product_set_delete_2
-    product_create_2 >> product_get_2 >> product_update_2 >> product_delete_2
+    product_set_create_2 >> product_set_create_2_idempotence >> product_set_delete_2
+
+    # ReferenceImage path
+    product_create_2 >> reference_image_create_2 >> reference_image_create_2_idempotence >> product_delete_2
+
+    # Product/ProductSet path
+    add_product_to_product_set_2 >> remove_product_from_product_set_2
+    product_set_create_2 >> add_product_to_product_set_2
+    product_create_2 >> add_product_to_product_set_2
+    remove_product_from_product_set_2 >> product_set_delete_2
+    remove_product_from_product_set_2 >> product_delete_2
+
+    # ####################### #
+    # ### No IDs examples ### #
```

Comment:
I'm not sure this is correct. The "No IDs examples" suggests this section is related to the previous "autogenerated" and "explicit id" sections, but this is rather a section with different types of operators.
Maybe we should rename it to sth like:
```suggestion
    # ### Annotate image examples ### #
```
if we want to separate this section.

Reply:
I apply your suggestion.